File: net/delivery-api/GetElement.cs
// DocSection: delivery_api_get_element
// Tip: Find more about .NET SDKs at https://kontent.ai/learn/net
using Kontent.Ai.Delivery;

// Creates an instance of the delivery client
// ProTip: Use DI for this in your apps https://kontent.ai/learn/net-register-client
IDeliveryClient client = DeliveryClientBuilder
      .WithProjectId("<YOUR_PROJECT_ID>")
      .Build();

// Gets the model of specific element within a specific content type
DeliveryElementResponse response = await client.GetContentElementAsync("article", "title");
IContentElement element = response.Element;
// EndDocSection

File: ruby/delivery-api/GetElement.rb
# DocSection: delivery_api_get_element
# Tip: Find more about Ruby SDKs at https://kontent.ai/learn/ruby
require 'delivery-sdk-ruby'

delivery_client = Kontent::Ai::Delivery::DeliveryClient.new project_id: '<YOUR_PROJECT_ID>'
delivery_client.element('article', 'title').execute do |response|
  ele = response.element
end
# EndDocSection